qcodes/dataset/plotting.py

```diff
@@ -36,22 +37,13 @@
 log = logging.getLogger(__name__)
 DB = qc.config["core"]["db_location"]
 
-AxesTuple = Tuple[Axes, Colorbar]
-AxesTupleList = Tuple[List[Axes], List[Optional[Colorbar]]]
-# NamedData is the structure _get_data_from_ds returns and that plot_by_id
-# uses internally
-NamedData = List[List[DSPlotData]]
+if TYPE_CHECKING:
+    AxesTuple = Tuple[Axes, Colorbar]
```

Comment:
These typevars cannot be inside `if TYPE_CHECKING` but you can use "" around the imports